qutebrowser/mainwindow/mainwindow.py

```diff
@@ -141,6 +140,24 @@ def on_config_changed(self, section, option):
         """Resize the completion if related config options changed."""
         if section == 'completion' and option in ('height', 'shrink'):
             self.resize_completion()
+        elif section == 'ui' and option == 'downloads-position':
+            self._add_widgets()
+
+    def _add_widgets(self):
```

Comment:
Can you add a small docstring here?

(I know the docstring checker is rather annoying... but it helps me when trying something and then accidentally forget writing docstrings, so I still think it's better to have a docstring for everything)